helm/hpcc/templates/thor.yaml

```diff
@@ -226,28 +228,31 @@ data:
 {{- end }}            
           {{- include "hpcc.addImagePullSecrets" . | nindent 10 -}}
           containers:
-{{- if $misc.postJobCommandViaSidecar }}
-{{ include "hpcc.addWaitAndRunContainer" (. | merge (dict "command" $misc.postJobCommand)) | indent 10 }}
+{{- range $containerNum := untilStep 1 (int (add1 $numWorkersPerPod)) 1 }}
+{{- $slavePort := add 20100 (mul 100 (sub $containerNum 1)) -}}
+{{- if and ($misc.postJobCommandViaSidecar) (eq $containerNum 1) }}
```

Comment:
This is a bit of a kludge, it saying only generate the postJob command for the 1st container.
Only 1 needs (on exit) to tell the pod sidecar (when this feature is on) to terminate.